deploy/config

```diff
@@ -41,10 +41,10 @@ eventstoreContainer_minor="0"
 eventstoreContainer="weareadaptive/eventstore:$eventstoreContainer_major.$eventstoreContainer_minor"
 
 # node
-vNode="6.2.1"
-nodeContainer_major="6"
-nodeContainer_minor="2"
-nodeContainer="weareadaptive/node:$nodeContainer_major.$nodeContainer_minor"
+vNode="8.11.3"
+nodeContainer_major="8"
+nodeContainer_minor="11.3"
+nodeContainer="node:$nodeContainer_major.$nodeContainer_minor"
```

Comment:
I'm not sure what the best practice for this is in Docker, but you might be able to just pull `node:8`.
This would keep us up to date with bug fixes and features without breaking the API.

Reply:
It makes a lot of sense, LTS will grant bug fixes, as long as we don't break the API I'm happy by going node:8.

Reply:
@tcdev00 this have been tested/validated ?

Reply:
yes @tdeheurles , I have tested node:8 in local and in circleci and works pretty well.
Initially I tried to use an alpine image but it was failing in both ends. The non alpine version works properly.